### PR TITLE
fix(ux): responsive ui display

### LINKED ISSUE #6647: The operation in SwaggerUIBundle is not responsive
I am using SwaggerUIBundle, everything works fine but I've noticed the operation is not responsive. You can see it when you inspect and switch it to mobile with vertical (tall) view:
image

The mobile horizontal (wide) view looks OK:
image

It would be nice if it can get rendered similar like the actual swagger-ui.html:
image

### PR DESCRIPTION
**Pull request by yzhe819**
### Description

Fixed link display on small screens
Also fixed the display issue of input element

### Motivation and Context

Fixes  #6647  #7338

### How Has This Been Tested?

Only update the relevant CSS

### Screenshots (if appropriate):

**The display issue:**

**After fixed:**

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by yzhe819** [comment]
I also tried to fix display issues with input elements. Related to #7338

Here is the result.
<details>
<summary>screenshot of issue</summary>


</details>

<details>
<summary>Fixed one</summary>


</details>

**Comment by yzhe819** [comment]
Hey @tim-lai! This pr is ready to be merged. Feel free to take a look.

**Comment by tim-lai** [comment]
@yzhe819 Thanks for the PR! It looks good and is now merged. Thanks for the contribution!

File: src/style/_layout.scss [hunk 1]
```diff
@@ -106,6 +106,18 @@
         flex: 1 1 150px;
         font-weight: 400;
     }
+
+    @media (max-width: 640px) {
+      small
+      {
+        flex: 1;
```
Comment (Saibamen) [inline src/style/_layout.scss:113]: Indenation

**Pull request merged**